src/app.d.ts

```diff
@@ -52,6 +52,19 @@ declare namespace App {
         cells?: GridEntry[];
     }
 
+    interface CollectionEntry {
+        id: string;
+        name: string;
+        singlePane: boolean;
```

Comment:
A comment about what `singlePane` means might be helpful. I wasn't sure at first.